### ISSUE TITLE
Python error: Py_Initialize: unable to load the file system codec

### ISSUE THREAD

**Issue by leenbzoold** [alicevision/meshroom#265]
I downloaded Meshroom-2018.1.0-win64.zip on Windows 10, unzipped it, clicked on the Meshroom executable and i only get these two error messages in the console:
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'
This happened while i had python 3.7.0 installed and it happened while i hadn't had it installed.
I've already checked the enviromental variables thingy. There is only Python37-32.

What do i do

**Comment by pixerati** [comment]
Are you trying to run via Remote Desktop?

**Comment by leenbzoold** [comment]
> Are you trying to run via Remote Desktop?

No, that was not what i was doing 6 months ago. I just tried to get it running directly on my machine (Windows 10 Pro).
I researched this python error online for a bit, tried to apply some of the solutions that i found, but nothing worked. So eventually i had to switch to the linux version of meshroom which ran without issues.
However though, my interest in this software (and in photogrammetry in general) then only lasted for about another month.
